### PR TITLE
fix: Social Icons for Dark Mode

### LINKED ISSUE #45: Social icons look weird in dark mode
Screen Shot 2022-01-14 at 07 11 54

Consider using `dark-only` and `light-only` classes. As they're recently merged into `main`, you might wish to branch again (if you're starting from scratch) or merge `main` to your existing branch to make these classes available.

### PR DESCRIPTION
**Pull request by anubhav06**
## Issue 
Fixes #45 

## About
Changed the styling of GitHub and Resume icon for dark mode <br>

Dark Mode: <br>
darkmode

Light Mode: <br>
lightmode

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

### CODE REVIEW

**Comment by VaibhaveS** [comment]
Hey this looks perfect! 
However i see some changes in the coordinates at map.js, was this intentional?

**Comment by anubhav06** [comment]
> Hey this looks perfect! However i see some changes in the coordinates at map.js, was this intentional?

Oh no no. I was figuring out how the map system works and I think accidently I committed the new location also. I'll remove that if you say

**Comment by CIPHERTron** [comment]
> Oh no no. I was figuring out how the map system works and I think accidently committed the new location also. I'll remove that if you say

That would be great. Please undo the changes

File: assets/js/map.js [hunk 1]
```diff
@@ -127,7 +127,7 @@ var fellows = [
         linkedin: 'https://www.linkedin.com/in/aniket-pal/',
         role: 'Fellow',
     },
-    {
+    {
```
Comment (ditsuke) [inline assets/js/map.js]:
```suggestion
    {
```

Comment (anubhav06) [reply assets/js/map.js]: Already done that 😀Maybe refresh for it to update

Comment (ditsuke) [reply assets/js/map.js]: unwanted whitespace addition, committing! 😄

Comment (ditsuke) [reply assets/js/map.js]: whoops! 😂

**Comment by VaibhaveS** [comment]
LGTM

**Comment by harshsharma6401** [comment]
Good Job with handling hover effects !

**Review by yyjlincoln — APPROVED** [review]
LGTM!

**Review by CIPHERTron — APPROVED** [review]
LGTM 🎉

**Comment by CIPHERTron** [comment]
Merging this PR since it has 3 approvers.

**Pull request merged**